src/modules/packages/main.py

```diff
@@ -114,6 +114,16 @@ def remove(self, pkgs):
             list of package names
         """
         pass
+    
+    @abc.abstractmethod
+    def remove_deps(self, pkgs):
```

Comment:
This abstract method is only implemented by a single package manager class.  What happens when someone who isn't using that package manager tries to use that functionality?

Reply:
True. I will remove it from here since what we need is already defined in Pacman class.